core/extensions/COMServerDemo/ReadMe.md

```diff
@@ -6,7 +6,7 @@ This is a basic example of providing a managed COM server in .NET Core 3.0. Docu
 Key Features
 ------------
 
-Demonstrates how to provide a COM server in .NET Core 3.0 Preview 5 or later.
+Demonstrates how to provide a COM server in .NET Core 3.0 Preview 8 or later.
 
 Additional comments are contained in source and project files.
 
```

Comment:
On line 16, it would be good to include a sentence about registered COM and regfree COM (for example, "You can build and execute the example either by registering the executable as a COM server or by using registration-free COM.").